references/searchable_attributes.md

```diff
@@ -2,8 +2,7 @@
 
 _Child route of the [settings route](/references/settings.md)._
 
-The values of the fields whose attributes are added to the searchable-attributes list are **searched for matching query words**.
-By default, all fields are considered to be `searchableAttributes`. This behavior is represented by the `["*"]` in the settings. Setting `searchableAttributes` to an empty array `[]` will reset the setting to its default value.
+The values associated with attributes in the `searchableAttributes` list are **searched for matching query words**. The order of the list also determines the [attribute ranking order](/guides/main_concepts/relevancy.md#attribute-ranking-order).
```

Comment:
Should we add something like `at its default value of "*", the order of attributes are determined by their order of appearance in the first document added`? But maybe it is not the place to do so.

Reply:
Great question. **For now**, I don't think we should add it. My goal at the moment is to make a precise distinction between references and other resources. Since this is an API reference for altering the index settings, I ideally want it to **only contain information related to that action**.

We might end up adding a line like this later. The behavior is confusing, so it might be useful to echo this explanation in as many places as possible. Personally, though, I think we should change the confusing behavior itself, instead of compensating with the docs.

Reply:
@qdequele